packages/frontend/src/services/recipesService.ts

```diff
@@ -1,7 +1,21 @@
 import { AxiosResponse } from "axios";
 import { ApiEndpoints } from "src/enums/apiEndpoints.enum";
-import { IRecipe } from "../../../shared/src/types/recipe.type";
+import { RecipesResponse } from "../../../shared/src/types/recipe.type";
 import { apiService } from ".";
 
-export const getAllRecipes = (): Promise<AxiosResponse<IRecipe>> =>
-  apiService.get(ApiEndpoints.Recipes);
+export const getAllRecipes = (
```

Comment:
lepiej przekazać tutaj obiekt z tymi wartościami, tak jak komuś pomyli się kolejność to będzie lipa